firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -821,19 +815,12 @@ static void check_rx_timeouts( void )
 // the setup routine runs once when you press reset:
 void setup( void ) 
 {
-
-    // duty Scalers good for 0x05
-    //SLADutyMax = 50;
-    //SLADutyMin = 5;
-    //SLRDutyMax = 50;
-    //SLRDutyMin = 20;
-
-
     // duty Scalers good for 0x02
```

Comment:
The comment should reflect that. 

"set the duty scalers to match the refresh rate of the PWMs, in this case 0x02"

No one will need to know what the specific value is, if they know what it's used for.